package RPC;

option cc_generic_services = true;

message RpcRequestData {
	required string service_name = 1;
	required uint32 call_id = 2;

	required bytes content = 3; 
}

message RpcResponseData {
	required uint32 call_id = 1;
	required uint32 err = 2;
	required bytes content = 3;
}



